flow-typed/react-intl_vx.x.x.js

```diff
@@ -75,8 +74,7 @@ declare module 'react-intl' {
     | 'textComponent'
     | 'defaultLocale'
     | 'defaultFormats'
-    | 'onError',
-  >;
+    | 'onError', >;
```

Comment:
Huh, weird. But I reproduce that that's what auto-formatting does... and this is not code we spend a lot of time in or really fully maintain, so probably better to just let it do as it likes here rather than try to persuade it to do something else. :shrug: